docs/source/en/model_doc/gptj.mdx

```diff
@@ -105,6 +105,30 @@ model.
 >>> gen_text = tokenizer.batch_decode(gen_tokens)[0]
 ```
 
+## Resources
+
+A list of official Hugging Face and community (indicated by 🌎) resources to help you get started with GPT-J. If you're interested in submitting a resource to be included here, please feel free to open a Pull Request and we'll review it! The resource should ideally demonstrate something new instead of duplicating an existing resource.
+
+<PipelineTag pipeline="text-generation"/>
```

Comment:
I think you need to add a blank line after the pipeline tag for it to render properly.

Reply:
Fixed